src/main/java/bio/terra/service/resourcemanagement/exception/BufferServiceAPIException.java

```diff
@@ -11,13 +11,19 @@ public class BufferServiceAPIException extends ErrorReportException {
 
   public BufferServiceAPIException(ApiException bufferException) {
     super(
-        "Error from Buffer Service: ",
+        "Error from Buffer Service",
         bufferException,
         Collections.singletonList(bufferException.getResponseBody()),
         HttpStatus.resolve(bufferException.getCode()));
     this.apiException = bufferException;
   }
 
+  // Used to reconstruct error message when reading job result from stairway
+  public BufferServiceAPIException(String message) {
+    super(message);
+    this.apiException = null;
+  }
```

Comment:
Ideally, we would prefer a constructor that accepts (String message, List<Throwable> causes) to match the DataRepoException case in the StairwayExceptionSerializer, but as far as I can tell, `ErrorReportException` doesn't support just those two arguments.

Reply:
That's a great idea - I've been wanting to work towards updating the serializer to surface more exception details and this is a great start.